ion-schema-cli

```diff
@@ -1,2 +1,7 @@
-#!/usr/bin/env sh
-./gradlew cli:install --quiet && exec cli/build/install/ion-schema-cli/bin/ion-schema-cli "$@"
+#!/usr/bin/env bash
+if [ -t 0 ]; then
```

Comment:
I'll add comments for these. I have no idea if `DATA=$(cat)` is an idiom—it was just a solution that worked to prevent any data being piped in from going to `./gradlew` instead of to ion-schema-cli.